pubsub/subscriptions/update_dead_letter.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package subscriptions
+
+// [START pubsub_update_dead_letter]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/pubsub"
+)
+
+func updateDeadLetter(w io.Writer, projectID, subID string, deadLetterTopicName string) error {
+	// projectID := "my-project-id"
+	// subID := "my-sub"
+	// deadLetterTopicName := "projects/my-project/topics/my-dead-letter-topic"
```

Comment:
Prefer `deadLetterTopicID := "my-dead-letter-topic"` and constructing `deadLetterTopic` before use in Line 38.

Reply:
Addressed in comments above.